libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -880,6 +880,44 @@ namespace SohImGui {
                     EnhancementSliderInt("King Zora Speed: %dx", "##WEEPSPEED", "gMweepSpeed", 1, 5, "");
                     EnhancementSliderInt("Vine/Ladder Climb speed +%d", "##CLIMBSPEED", "gClimbSpeed", 0, 12, "");
 
+                    const char* autotunic_cvar = "gAutoTunics";
+                    {
+                        int val = CVar_GetS32(autotunic_cvar, 0);
+                        val = MAX(MIN(val, 3), 0);
+                        int fps = val;
+
+                        if (val == 0)
+                        {
+                            ImGui::Text("Tunic Upgrades: Off");
+                        }
+                        else if (val == 1)
+                        {
+                            ImGui::Text("Tunic Upgrades: Auto-Switch");
+                        }
+                        else if (val == 2)
+                        {
+                            ImGui::Text("Tunic Upgrades: Permanent Upgrade");
+                        }
+                        else if (val == 4)
```

Comment:
Should be val == 3 based on the slider limits below?

Reply:
Yep, don't know how I managed that one. Thanks again!